tsl/test/t/003_connections.pl

```diff
@@ -66,3 +66,75 @@
 is( $stdout,
 	q[2021-09-19 18:00:00|1|23.4
 dn1|alice|f]);
+
+# Test DROP OWNED BY and REASSIGN OWNED BY behavior across the setup
```

Comment:
Seems odd to put this in the "connections" TAP test. Is this intentional?

Reply:
Users involves, setting/resetting sessions. Didn't want to create a new tap test :-)

Reply:
Yeah, I think that is stretching it. Maybe this is fine for now, but don't want to add too much to tech debt in case we start adding a ton of unrelated stuff and building monolithic tests.

Reply:
I have renamed the test to `003_connections_privs.pl` because that is what it is testing really.